addoption-call.md

```diff
@@ -0,0 +1,88 @@
+The incentives to ensure CT logs are well-operated and "checked" are enormous.
+
+For transparency systems involving application data or general transparency, the value of the data assured is much more subjective, depending on individual publishers and consumers.
+In heterogeneous situations, such as digital supply chains, the incentives to check and audit are much less clear.
+
+Yet, transparency logs are trustworthy only when they are independently checked and monitored.
+
+I believe that by specifying a verifiable data structure that minimizes the costs and complexities of replicating, auditing, and verifying, adoption bariers for ransparency techniques are usefuly reduced.
+This is especially so in "just in case" situations where the value often becomes evident when it’s already too late.
+
+In the draft, I've highlighted the properties of MMRs that, in my view, have led to their wide adoption in systems that benefit from a highly replicable, prunable, and permanently stable VDS.
+
+The implementation section includes references (though not exhaustive) to several production implementations, with broad similarities between them.
+Trigger warning: most of these are "blockchains," but that seems to be fertile ground for testing many useful techniques, and there is certainly a strong incentive to attack them (double meaning intended).
+
+Independent of blockchains, the draft leans heavily on work by Leonid Reyzin and Sophia Yakoubov, particularly their paper on "Efficient Asynchronous Accumulators for Distributed PKI" -- https://eprint.iacr.org/2015/718.pdf.
+
+Their work provides a formal model that applies to the emergent benefits of MMRs, which is particularly important for the formulation of proofs of inclusion and consistency specified in the draft.
+
+I should note that while I’ve made Leonid and Sophia aware of this draft, and they confirmed that the basic link between MMRs and their work makes sense, they have not reviewed the draft or otherwise endorsed it.
+
+On a more personal note, this draft is the one I wish had been available when I first became convinced that MMRs were a great fit for more application-centric, "eye of the beholder-valued" transparency use cases.
+It is also the result of poring over various implementations and considering which, if any, differences mattered.
+
+Here are 6 properties that I think particularly help lower the barrier to ensuring trustworthy transparency systems:
+
+1. Immediate replication after element addition is safe; the log data does not change after addition.
+
+While regular checkpoints and consistency proofs can be requested for any Merkle-based log, being able to instantly replicate the VDS data puts the verifier in control of the frequency and granularity of verification.
+Since each replicated (and attested) log is as reliable as the original, this can also significantly improve scalability.
```

Comment:
I believe you're talking about the MMR comparison, which wasn't as obvious. Suggest you include "MMR", to remove ambiguity of which "this" or assumption of which you're referring to
```suggestion
Since each replicated (and attested) MMR log is as reliable as the original, <this> can also significantly improve scalability.
```